cmd/root.go

```diff
@@ -31,6 +33,7 @@ func setupLogger() *zap.Logger {
 	logCfg.EncoderConfig.EncodeTime = customTimeEncoder
 	if debugMode {
 		go func() {
+			defer sentry.Recover()
```

Comment:
No, mentioning sentry in every goroutine is done just because since goroutines are running concurrently each goroutine needs to take care of its sentry related data.